README.md

```diff
@@ -77,4 +77,4 @@ public class Chat : NetworkedBehaviour
 ```
 
 ### License
-[MIT Licence](LICENSE)
\ No newline at end of file
+[MIT License](LICENSE)
```

Comment:
the original file this link points towards is still `LICENCE` not `LICENSE` — do you mind including that file rename change into this PR as well?

Reply:
Done